src/parser.ts

```diff
@@ -92,6 +97,21 @@ export class TrailingCommaError implements SourceError {
   }
 }
 
+export function parseAt(source: string, num: number) {
+  let program: es.Program | undefined
+  try {
+    program = (acornParseAt(source, num) as unknown) as es.Program
```

Comment:
This comment is directed at me so don't take it personally.

https://github.com/acornjs/acorn/blob/f164d84661f4f79486c84a4c0d1ca2233614c088/acorn/dist/acorn.d.ts#L7

parseExpressionAt and parse returns acorn.Node, not es.Program which is why there's this weird cast to unknown. I understand you just copied it from what I wrote in parse below, so I'll fix that later.

Please check the return types of imported functions next time.